server/scsynth/scsynth_main.cpp

```diff
@@ -359,8 +360,7 @@ int main(int argc, char* argv[]) {
     }
     fflush(stdout);
 
-    World_WaitForQuit(world, true);
-
+    EventLoop::run([&]() { World_WaitForQuit(world, true); });
```

Comment:
explicit capture please (i have also added a note on this to the wiki style guide, since this is not the first review where it's come up)